build.gradle

```diff
@@ -32,7 +34,7 @@ if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
 
 allprojects {
   group = "com.netflix.iceberg"
-  version = '0.3.1'
+  version = gitVersion()
```

Comment:
How is the version determined?

Reply:
Per https://github.com/palantir/gradle-git-version

It should match the output of `git describe --tags --always --first-parent`. If the repository is dirty it appends `.dirty`.

Reply:
I was hoping to have a more clear explanation, like given the tag 0.3.2, this would produce 0.3.3 or something. Also, would this affect the JitPack builds based on tags?

Reply:
Sorry, should have been clearer here.

It takes the current tag and appends the number of commits on top of that tag and the git hash. If one were to check out my fork on this branch, and run `./gradlew printVersion`, it returns `0.3.2-5-g18f87a8`. The 5 is for there being 5 commits since 0.3.2 - the merged PR #68 . and the 4 commits that comprise this branch.

This shouldn't affect the JitPack builds, as soon as we tag a commit the `gitVersion()` resolves to that tag.

Reply:
Awesome, thanks for the explanation!